### PR TITLE
sage

### PR DESCRIPTION
**Pull request by odsod**
- chore(deps): bump dependencies
- ci: migrate to Sage

### CODE REVIEW

File: .sage/proto.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+		"-o",
+		sg.FromBinDir("protoc-gen-bq-json-schema"),
+		"./protoc-gen-bq-json-schema",
+	).Run()
+}
+
+func (Proto) BufGenerate(ctx context.Context) error {
+	sg.Deps(ctx, Proto.ProtocGenGo, Proto.ProtocGenBigqueryJSONSchema)
+	sg.Logger(ctx).Println("generating example proto stubs...")
+	cmd := sgbuf.Command(ctx, "generate", "--template", "buf.gen.yaml", "--path", "einride")
```
Comment (ericwenn) [inline .sage/proto.go:54]: empty gen directory first

File: Makefile.old [hunk 2]
```diff
@@ -0,0 +1,47 @@
+SHELL := /bin/bash
```
Comment (ericwenn) [inline Makefile.old:1]: are these still needed?

File: .sage/proto.go (hunk 1 above)
Comment (odsod) [reply .sage/proto.go:54]: aye capn!

File: Makefile.old (hunk 2 above)
Comment (ericwenn) [reply Makefile.old:1]: this file I mean

**Review by ericwenn — APPROVED** [review]

File: Makefile.old (hunk 2 above)
Comment (odsod) [reply Makefile.old]: wopps! removed

**Pull request merged**